tests/TestData/HtmlOutput/qr-additional-information.png.print.html

```diff
@@ -160,31 +156,22 @@
     margin-top: -11pt;
 }
 
-#qr-bill-separate-info {
-    border-bottom: 0;
-}
-
 #qr-bill-separate-info-text {
     display: none;
 }
 
-#qr-bill-receipt {
-    border-right: 0;
-}
+
 </style>
 
 <table id="qr-bill">
     <tr id="qr-bill-separate-info">
-        <td colspan="99"><span id="qr-bill-separate-info-text">Separate before paying in</span></td>
+        <td colspan="99"><span id="qr-bill-separate-info-text"></span></td>
```

Comment:
The separate-before-printing-text seems to have gotten lost.

Reply:
Oh, that seems like a bug. I'll investigate.

Reply:
Alright, just stumbled upon the why now. This is the print-version of the output, so there are no separators needed anyway, right?
Before my changes the text was still present here but the whole span `#qr-bill-separate-info-text` was just set to `display: none` in the print version. This is again easily possible with css, but harder with xml.
I would have had the option using `<fo:block visibility="hidden">` but that wouldn't have solved the border solution of the separator.
I solved this in \Sprain\SwissQrBill\PaymentPart\Output\MarkupOutput\AbstractMarkupOutput::translateContents
by doing:
```            if('separate' === $key && true === $this->isPrintable()){
                // Do not display the separator text at all when printable is true.
                $paymentPart = str_replace('{{ text.' . $key . ' }}', '', $paymentPart);
                continue;
            }
```
Just remove the tag so nothing gets placed anymore. I agree that this is neither the best place to do this nor the most beautiful way. But the alternative would have been to add another class similar to PrintableStylesTemplate.php  just for this text. This was some kind of lazy way around it that still worked with HtmlOutput still.

Reply:
The separate-before-printing-text is back in town! See commit https://github.com/sprain/php-swiss-qr-bill/pull/78/commits/ce2e3b16fe51979f73a094554bcf079ebda459cb

I removed that nasty part in `translateContents` and added another template that has the only job of hiding things in printable mode.